sema/subyt/j2/generator.py

```diff
@@ -0,0 +1,49 @@
+import os
+from typing import Callable
+
+from jinja2 import select_autoescape
+
+from sema.commons.j2 import Filters, J2RDFSyntaxBuilder
+from sema.subyt.api import Generator
+
+
+class JinjaBasedGenerator(Generator):
+    """
+    Core class for the jinja based LD Templating service.
+    """
+
+    def __init__(self, templates_folder: str = "."):
+        """
+        Builds the generator that produces LD output from datasources
+
+        :param templates_folder: Location of the templates defaults to "."
+        """
+        self._templates_folder = templates_folder
+        ttl_filter = {"ttl": Filters.all()["xsd"]}
```

Comment:
do we still want this?
move to sema looks like a good time to get rid off the older ttl syntax?

Reply:
I have no idea what this does so I'm leaving it here for now

Reply:
this keeps `| ttl` working as it looks up the xsd filter and registers it again as ttl too